cdp_scrapers/legistar_utils.py

```diff
@@ -1310,6 +1313,99 @@ def get_content_uris(self, legistar_ev: Dict) -> List[ContentURIs]:
         )
         raise NotImplementedError
 
+    def inject_known_person(self, person: Person) -> Person:
+        """
+        Inject information from self.known_persons
+        if person exists in the dictionary for long-term static data
+
+        Parameters
+        ----------
+        person: Person
+            Person into which to inject data from known_persons
+
+        Returns
+        -------
+        Person
+            Input person updated with more information
+            or as-is if person not found in known_persons
+        """
+        try:
+            known_person = self.known_persons[person.name]
+        except (TypeError, KeyError):
+            return person
+
+        for attr in [
+            "email",
+            "is_active",
+            "name",
+            "phone",
+            "picture_uri",
+            "router_string",
+            "seat",
+            "website",
+        ]:
+            # input person has new information so prefer that
+            # over static long-term information
+            setattr(person, attr, getattr(person, attr) or getattr(known_person, attr))
+
+        # now that we have seat from static hard-coded data
+        # we can bring in seat.roles (OfficeRecords from Legistar API)
+        if person.seat and not person.seat.roles:
+            person.seat.roles = self.get_roles(
+                get_legistar_person(
+                    client=self.client_name,
+                    person_id=person.external_source_id,
+                    use_cache=True,
+                )[LEGISTAR_PERSON_ROLES]
+            )
```

Comment:
After finishing this pull request, will probably move this blob for `Person.seat.roles` into post-processing phase described in #31 . Think it is more compatible with post-processing.

Reply:
Agree but nice job none-the-less